docker/lr-winnowmap/Makefile

```diff
@@ -0,0 +1,14 @@
+VERSION = 2
+#TAG1 = quay.io/broad-long-read-pipelines/lr-canu:$(VERSION)
```

Comment:
Usually, we take out commented-out code before PRs, so if this code is no longer needed, they should be just removed.